src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/RaCandidateProjector.php

```diff
@@ -49,6 +56,10 @@ public function __construct(RaCandidateRepository $raCandidateRepository)
      */
     public function applySecondFactorVettedEvent(SecondFactorVettedEvent $event)
     {
+        if ($this->raListingRepository->findByIdentityId($event->identityId)) {
+            return;
```

Comment:
Very unfortunate we have to introduce a time coupling with another read model, but I don't see another solution, currently. Perhaps it would be wise to start adding the identity's RA status to the events pertaining to RA candidate nomination so we can depend on that information, but I'll have to think about that.